src/pnm/encoder.rs

```diff
@@ -626,28 +626,3 @@ impl<'a> TupleEncoding<'a> {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
This reminds me: We should think about splitting `pnm` into a seperate crate where we can support the more arcance details, similar to `png` etc. Please remind me if I forget ;)